examples/oneview_logical_enclosure.yml

```diff
@@ -29,8 +30,7 @@
         config: "{{ config }}"
```

Comment:
Can you please replace this line# 19 with actual values like below:
   hostname: 172.16.101.48
   username: administrator
   password: my_password   
   api_version: 600
In [PR#306](https://github.com/HewlettPackard/oneview-ansible/pull/306), Felipe has commented that it has been requested by Ansible so that we can keep our documentation synced with the modules in the ansible core repository.